eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -287,6 +286,27 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <module name="MissingJavadocType"/>
     <module name="MissingJavadocPackage"/>
 
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="expected"/>
```

Comment:
I probably wouldn't enforce a name like this. `expected` doesn't read well when the exception is not expected!